// @copilotkit/react-ui/dist/components/index.mjs
import "../chunk-MMVDU6DF.mjs";
import "../chunk-SC6JRFAJ.mjs";
import "../chunk-WB3YULQ4.mjs";
import {
  CopilotPopup
} from "../chunk-QEZ3YYLO.mjs";
import {
  CopilotSidebar
} from "../chunk-JU3HNL5N.mjs";
import "../chunk-FWKHST6I.mjs";
import "../chunk-TIDV74OE.mjs";
import "../chunk-UH2UFL5W.mjs";
import "../chunk-V7W6IM2V.mjs";
import {
  CopilotDevConsole
} from "../chunk-VGPQYMKJ.mjs";
import "../chunk-Q5V6S67N.mjs";
import {
  shouldShowDevConsole
} from "../chunk-6TCUJ3B7.mjs";
import "../chunk-KXE2JCUH.mjs";
import "../chunk-NRA3CFEE.mjs";
import "../chunk-BH6PCAAL.mjs";
import "../chunk-UFN2VWSR.mjs";
import {
  CopilotChat
} from "../chunk-IAHSHPRJ.mjs";
import "../chunk-MPZAMNCN.mjs";
import "../chunk-ALWZ35SI.mjs";
import {
  RenderImageMessage
} from "../chunk-UKWBNVMQ.mjs";
import "../chunk-6VUDBDUN.mjs";
import "../chunk-5INP2EWU.mjs";
import {
  AssistantMessage
} from "../chunk-XMNTLIK5.mjs";
import {
  UserMessage
} from "../chunk-HWMFMBJC.mjs";
import {
  Suggestions
} from "../chunk-QGSPTXOV.mjs";
import {
  Suggestion
} from "../chunk-IMBPSLL4.mjs";
import "../chunk-PLHTVHUW.mjs";
import "../chunk-EMIYIMQ6.mjs";
import "../chunk-CGEAG65D.mjs";
import "../chunk-YQFVRDNC.mjs";
import {
  Markdown
} from "../chunk-YTXEWDNC.mjs";
import "../chunk-KENCH7RN.mjs";
import "../chunk-B3D7U7TJ.mjs";
import {
  useChatContext
} from "../chunk-IEMQ2SQW.mjs";
import "../chunk-XWG3L6QC.mjs";
import "../chunk-54JAUBUJ.mjs";
import "../chunk-JGMFJZMG.mjs";
import "../chunk-S5MBUNGN.mjs";
import "../chunk-MRXNTQOX.mjs";
export {
  AssistantMessage,
  CopilotChat,
  CopilotDevConsole,
  CopilotPopup,
  CopilotSidebar,
  Markdown,
  RenderImageMessage,
  Suggestion as RenderSuggestion,
  Suggestions as RenderSuggestionsList,
  UserMessage,
  shouldShowDevConsole,
  useChatContext
};
